src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,77 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+1.17.0.dev0 (5/13/2019)
+-----------------------
+
+New Features
+~~~~~~~~~~~~
+
+* V2 `./pants test.pytest` selects interpreter based off of compatibility constraints (#7679)
```

Comment:
@stuhood is this a bug fix or new feature? It is new behavior, but really it's playing catch up to V1 `test.pytest` so in some ways is fixing things?

Reply:
Either bugfix or refactor probably. But there is minimal value in being super precise in the categorization of release notes currently.